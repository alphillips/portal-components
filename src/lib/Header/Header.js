import React from "react";

import './header.css'

const styles = {
  fontWeight:'800'
}

const Header = () => (
  <div className={'header-main'} style={styles}>
    This is the header!!
  </div>
);

export default Header;
