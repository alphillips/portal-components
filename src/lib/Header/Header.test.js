import React from "react";
import Header from "./Header";
import renderer from "react-test-renderer";

describe("Header", () => {
  it("renders properly", () => {
    const tree = renderer
      .create(<Header />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
