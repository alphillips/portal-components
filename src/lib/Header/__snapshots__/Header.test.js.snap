// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders properly 1`] = `
<div>
  This is the header!
</div>
`;
